platforms/documentation/docs/src/docs/userguide/authoring-builds/tasks/custom_tasks.adoc

```diff
@@ -554,6 +554,40 @@ Worker daemons will remain running until either the build daemon that started th
 In order to support cancellation (e.g. when the user stops the build with CTRL+C) and task timeouts, custom tasks should react to their executing thread being interrupted.
 The same is true for work items submitted via the worker API. If a task does not respond to an interrupt within 10s, the daemon will shut down in order to free up system resources.
 
+== Verification Failures
+
+Exceptions thrown during task execution result in a failure that immediately terminates a build.
+The result of the build will be `FAILED`, and no further tasks will be executed.
+When <<command_line_interface.adoc#sec:continue_build_on_failure, running with the `--continue` flag>> Gradle will continue to run other requested tasks in the build after encountering a task failure.
+However, any tasks that depend on a failed task will be not be executed.
+
+There is a special type of exception that behaves differently.
+A task can throw a subtype of link:{javadocPath}/org/gradle/api/tasks/VerificationException.html--[VerificationException] to indicate that it has failed in a controlled manner such that its output is still valid for consumers.
```

Comment:
The `VerificationException` [link](https://builds.gradle.org/repository/download/Gradle_Release_Check_BuildDistributions/76138450:id/distributions/gradle-8.6-docs.zip!/gradle-8.6-20231220211144%2B0000/docs/javadoc/org/gradle/api/tasks/VerificationException.html--) didn't work for me in the [build](https://builds.gradle.org/repository/download/Gradle_Release_Check_BuildDistributions/76138450:id/distributions/gradle-8.6-docs.zip!/gradle-8.6-20231220211144%2B0000/docs/userguide/custom_tasks.html#verification_failures)

Reply:
Got it.